frontend/packages/kubevirt-plugin/src/components/create-vm/hooks/use-vm-templates-resources.tsx

```diff
@@ -22,6 +23,10 @@ export const useVmTemplatesResources = (namespace: string): useVmTemplatesResour
     },
     isList: true,
   });
+  const [dataSources, dataSourceLoaded] = useK8sWatchResource<DataSourceKind[]>({
```

Comment:
Very minor nitpick: `dataSourceLoaded` -> `dataSourcesLoaded`
This can be fixed in the followup PR, though.

Reply:
fixed, nice catch :)